src/api_wrappers/generatePythonAPIWrappers.py

```diff
@@ -109,8 +109,8 @@ def camel_case_to_underscore(name):
 for method in json.loads(sys.stdin.read()):
     route, signature, opts = method
     wrapper_method_name = camel_case_to_underscore(signature.split("(")[0])
-    retry = "True" if (opts['retryable']) else "False"
-    accept_nonce = True if 'acceptNonce' in opts else False
+    accept_nonce = opts.get('acceptNonce', False)
+    retry = opts['retryable'] or accept_nonce
```

Comment:
The logic makes sense.  Is there a case where a call is not retryable but does accept a Nonce, or is this more of a historical artifact? https://github.com/dnanexus/dx-toolkit/blob/master/src/api_wrappers/wrapper_table.json#L533